sdk/storage/test/test_base.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "gtest/gtest.h"
+
+namespace Azure
+{
+namespace Storage
+{
+namespace Test
+{
+  struct TestUtility 
+  {
+    constexpr static const char* k_STANDARD_STORAGE_CONNECTION_STRING = "";
+    constexpr static const char* k_PREMIUM_STORAGE_CONNECTION_STRING = "";
+    constexpr static const char* k_BLOB_STORAGE_CONNECTION_STRING = "";
+    constexpr static const char* k_PREMIUM_FILE_CONNECTION_STRING = "";
+    constexpr static const char* k_ADLS_GEN2_CONNECTION_STRING = "";
+
+    // TODO: Temporary
+    constexpr static const char* k_ACCOUNT_SAS = "";
+  };
+}
+}
+}
```

Comment:
The last three `}` should be in the same line.

Reply:
Same with the last comment.